exercises/affine-cipher/.meta/hints.md

```diff
@@ -0,0 +1,11 @@
+## Advanced
```

Comment:
I would NOT add this file and just let the the readme match the description in the problem-specifications repository.

Reply:
It was a good idea to create the hints file to recreate the missing info though. :star: